chsdi/templates/htmlpopup/lubis.mako

```diff
@@ -114,7 +114,7 @@ viewer_url = get_viewer_url(request, params)
   </td>
 </tr>
 
-% elif c['attributes']['doi_link'] and c['attributes']['doi_link'].startswith('http'):
+% elif 'doi_link' in c['attributes'] and c['attributes']['doi_link'].startswith('http'):
```

Comment:
`if c['attributes'].get('doi_link', '').startswith('http'):` will do the same. also above to test if a key exists and if yes evaluate the truth value is simply `c['attributes'].get('doi_link', False)`

Reply:
thank you for that hint.